src/app/components/AudioVideoHead/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { arrayOf, shape, string } from 'prop-types';
+
+const AudioVideoHead = ({ audioVideoAssets }) => {
+  const settingsArray = audioVideoAssets.map(
+    ({ mediaPlayerSettings }) => mediaPlayerSettings,
+  );
+
+  // The [${settingsArray}] syntax is needed due to its usage in the string literal
+  const loadPlayers = `
+      var settings = [${settingsArray}];
+      ${audioVideoAssets
+        .map(
+          (avAsset, index) =>
+            `var player${index} = bump.player(document.getElementById('${avAsset.id}'), settings[${index}]);
+          player${index}.load();`,
+        )
+        .join(' ')}
+    `;
+
+  return (
+    <>
+      <Helmet>
+        <script type="text/javascript">
+          {`
+            function mediaPlayerSetup(container) {
+              require(['bump-4'], (bump) => {
+                ${loadPlayers}
+              });
+            }
+          `}
+        </script>
+        <script type="text/javascript">
+          {`
+            function initialiseRequires() {
+              var requiredScripts = { "bump-4": "https://emp.bbci.co.uk/emp/bump-4/bump-4" };
+              require({ paths: requiredScripts, waitSeconds: 30 });
```

Comment:
What happens if this times out? Will the user be shown a blank space where the video would've been shown?

Reply:
AFAICS there should be no blank space as we are not currently reserving space for the player, but the player will fail to load.